perf.groovy

```diff
@@ -260,15 +282,18 @@ def static getOSGroup(def os) {
             }
 
             def osGroup = getOSGroup(os)
-            def architecture = 'x64'
-            def configuration = 'Release'
             def runType = isPR ? 'private' : 'rolling'
             def benchViewName = isPR ? 'coreclr private \$BenchviewCommitName' : 'coreclr rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
 
             steps {
                 shell("./tests/scripts/perf-prep.sh")
                 shell("./init-tools.sh")
-                shell("./build.sh ${architecture} ${configuration}")
+                copyArtifacts(fullBuildJobName) {
```

Comment:
Makes sense, thanks for catching that! I'll make the change to match what we do in xunit-performance. The logic was ported from our flow job in optimization, so I think we want to apply a fix to do the same build-number-passing there as well.